.spelling

```diff
@@ -0,0 +1,49 @@
+APRILTAG
+BODMAS
+camcon
+DataTraveler
+Duratool
+DTSE9H
+Ebuyer
+imax
+Nikitin
+ODROID
+overcurrent
```

Comment:
Hrm, is this really a word? I'd be tempted to hyphenate this.

Reply:
It's got a [wikipedia](https://en.wikipedia.org/wiki/Overcurrent) and seems to be in a standard ([IEC 60364](https://webstore.iec.ch/preview/info_iec60364-1%7Bed5.0%7Den_d.pdf) 131.4) so think its ok to be a word.